tests/unit/local/test_internal.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from chalice import Chalice
+from chalice.local.internal import RequestHandler
+
+
+@pytest.fixture
+def sample_app():
```

Comment:
Can you add some additional test cases?

I'm thinking:

* One for returning a `Response()` object.
* One that has multi value headers (a value in the `headers` dict is a list).
* A test for a non 200 response.
* A test where the view function raises an unexpected exception.

Reply:
Added only test cases that you suggested,

- One for returning a Response() object.
  - https://github.com/aws/chalice/pull/1193/commits/4e413ee2dd1cd2e944c018109fc8a3138954974b
- One that has multi value headers (a value in the headers dict is a list).
  - https://github.com/aws/chalice/pull/1193/commits/838f061217f152c62c77e843209b93d20db12b5c
- A test for a non 200 response.
  - https://github.com/aws/chalice/pull/1193/commits/7d1873f55c2438f1cb23b6ea507f7ce282af4c1c
- A test where the view function raises an unexpected exception.
  - https://github.com/aws/chalice/pull/1193/commits/9b1aef71b7515b7d7e5e8e1954cef350c4583595